src/mongo-to-grpc-transformer.js

```diff
@@ -6,16 +6,17 @@ const qlik = require('./qlik_grpc');
 const GRPC_CHUNK_SIZE = 100;
 
 class MongoToGrpcTransformer extends stream.Transform {
-  constructor(call) {
+  constructor(call, booleanType) {
     super({ objectMode: true, writableObjectMode: true, readableObjectMode: true });
     this.call = call;
     this.headerSent = false;
     this.rows = [];
     this.fieldInfo = [];
+    this.booleanType = booleanType || 'numeric';
   }
 
   _buildFieldInfo(firstChunk) {
-    const fields = Object.keys(firstChunk).filter(name => typeof firstChunk[name] !== 'object').map(name => ({
```

Comment:
By removing this filter an additional column `_id` will be added to the data set, and this is what is failing the Circle CI test. Is that change intentional? If `_id` is needed we need to filter the string in the test case instead.

Reply:
Saw that you filed #66, so I guess this change is indeed intentional 👍 

Can you update the test case as well? The rest of the changes looks good to me.